pkg/intermediate/aggregate_test.go

```diff
@@ -52,18 +52,32 @@ var (
 		"packetDeltaCount",
 		"reversePacketTotalCount",
 		"reversePacketDeltaCount",
+		"octetTotalCount",
+		"reverseOctetTotalCount",
 	}
 	antreaSourceStatsElementList = []string{
 		"packetTotalCountFromSourceNode",
 		"packetDeltaCountFromSourceNode",
 		"reversePacketTotalCountFromSourceNode",
 		"reversePacketDeltaCountFromSourceNode",
+		"octetTotalCountFromSourceNode",
+		"reverseOctetTotalCountFromSourceNode",
 	}
 	antreaDestinationStatsElementList = []string{
 		"packetTotalCountFromDestinationNode",
 		"packetDeltaCountFromDestinationNode",
 		"reversePacketTotalCountFromDestinationNode",
 		"reversePacketDeltaCountFromDestinationNode",
+		"octetTotalCountFromDestinationNode",
+		"reverseOctetTotalCountFromDestinationNode",
+	}
+	antreaThroughputElements = []string{
+		"throughput",
+		"reverseThroughput",
+		"throughputFromSourceNode",
+		"throughputFromDestinationNode",
+		"reverseThroughputFromSourceNode",
+		"reverseThroughputFromDestinationNode",
```

Comment:
It would be better to group the stats by non-reverse and reverse like we did before.